example/optimus/coordinator/run

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -e
+
+trap "kill 0" EXIT INT TERM ERR
```

Comment:
don't need this bc there are no background processes, applies to all added scripts